src/App.jsx

```diff
@@ -14,33 +16,98 @@
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORT_BY_ALPHABIT = 'alphabet';
+const SORT_BY_LENGTH = 'length';
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+function getPreparedGoods(goods, { sortField, reverse }) {
+  let preparedGoods = [...goods];
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+  preparedGoods.sort((good1, good2) => {
+    switch (sortField) {
+      case SORT_BY_ALPHABIT:
+        return good1.localeCompare(good2);
 
-      <button type="button" className="button is-danger is-light">
-        Reset
-      </button>
-    </div>
+      case SORT_BY_LENGTH:
+        return good1.length - good2.length;
+
+      default:
+        return 0;
+    }
+  });
+
+  if (reverse) {
+    preparedGoods = preparedGoods.toReversed();
+  }
+
+  return preparedGoods;
+}
+
+export const App = () => {
+  const [sortField, setSortField] = useState('');
```

Comment:
The initial state for `sortField` is set to an empty string. Consider setting a default sort field, such as `SORT_BY_ALPHABIT` or `SORT_BY_LENGTH`, to ensure that the goods are sorted by default when the component is first rendered.